packages/core/src/components/alert/alert.tsx

```diff
@@ -21,57 +29,70 @@ export class AtomAlert {
 
   private handleClose = () => {
     this.atomClose.emit()
+    this.isAlertOpen = false
   }
 
   private handleAction = () => {
     this.atomAction.emit()
   }
 
+  @State() isAlertOpen = true
+
   render() {
     return (
-      <Host role='alert'>
-        <div
-          class={{
-            [`atom-alert`]: true,
-            [`atom-color--${this.color}`]: true,
-            [`has-close`]: this.close,
-          }}
-        >
-          <div class='atom-body'>
-            {this.icon && (
-              <atom-icon class='atom-icon' icon={this.icon}></atom-icon>
-            )}
-            <div class='atom-content'>
-              {this.messageTitle && (
-                <div class='atom-title' innerHTML={this.messageTitle}></div>
-              )}
-              {this.messageText && (
-                <div class='atom-message' innerHTML={this.messageText}></div>
+      <Host
+        role='alert'
+        aria-live='assertive'
+        aria-atomic='true'
+        aria-hidden={!this.isAlertOpen && 'true'}
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
**praise:** ‏:rocket: